tap_jira/streams.py

```diff
@@ -369,3 +379,7 @@ def validate_dependencies():
             errs.append(msg_tmpl.format("Issue Transitions", "Issues"))
     if errs:
         raise DependencyException(" ".join(errs))
+
+def transform_user_date(user_date):
+    """Transform date value to 'yyyy-mm-dd' format."""
```

Comment:
Are we changing the format here? If not then do mention what we are doing, and mention all language support things.